src/styles/top.scss

```diff
@@ -0,0 +1,62 @@
+body {
+  margin: 0;
+  font-family: $font-family;
+  scroll-behavior: smooth;
+}
+
+.header {
+  background: url('../images/background-thumb.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  padding-inline: $padding-mobile;
+
+  @include on-tablet {
+    padding-inline: 72px;
+  }
+
+  @include on-desktop {
+    padding-inline: 120px;
+  }
+
+  &__content{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100vh;
+  }
+
+  .top-bar {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-block: 36px;
+
+    &__icons {
+      display: flex;
+      gap: 24px;
+    }
+
+    &__logo-link {
+      display: flex;
+    }
+
+    &__logo {
+      height: 24px;
+
+      @include hover(transform, scale(1.1));
+    }
+  }
+
+  .bottom{
```

Comment:
Do not define a block in the context of another block [Reference](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#:~:text=Styling%20a%20Block%20in%20the%20Context%20of%20Another%20Block)